docs/package.json

```diff
@@ -30,17 +30,17 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.30",
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
     "@fortawesome/react-fontawesome": "^0.1.11",
-    "@material-ui/core": "5.0.0-alpha.38",
+    "@material-ui/core": "5.0.0-beta.0",
     "@material-ui/data-grid": "^4.0.0-alpha.29",
-    "@material-ui/docs": "5.0.0-alpha.37",
-    "@material-ui/icons": "5.0.0-alpha.37",
-    "@material-ui/lab": "5.0.0-alpha.38",
+    "@material-ui/docs": "5.0.0-beta.0",
+    "@material-ui/icons": "5.0.0-beta.0",
+    "@material-ui/lab": "5.0.0-beta.0",
     "@material-ui/styled-engine": "5.0.0-alpha.34",
     "@material-ui/styled-engine-sc": "5.0.0-alpha.35",
```

Comment:
These two packages should go to beta too, no?

Reply:
Yeah, I suppose. The versioning script didn't include them.